tools/staramr/staramr_search.xml

```diff
@@ -38,33 +39,36 @@
             --no-exclude-genes
         #end if
 
+        #if str($advanced.plasmidfinder_type.plasmidfinder_type_condition) != 'default'
+            --plasmidfinder-database-type '${advanced.plasmidfinder_type.plasmidfinder_type_condition}'
+        #end if
+
         --output-summary $summary
+        --output-detailed-summary $detailed_summary
         --output-resfinder $resfinder
-        #if $use_pointfinder.enable
+        --output-plasmidfinder $plasmidfinder
+        #if str($use_pointfinder.use_pointfinder_condition) != 'default'
             --output-pointfinder $pointfinder
         #end if
         --output-settings $settings
         --output-excel $excel_proper_extension
 
         --output-hits-dir staramr_hits
 
-        #if $use_pointfinder.enable
-            --pointfinder-organism $use_pointfinder.organism
+        #if str($use_pointfinder.use_pointfinder_condition) != 'default'
+            --pointfinder-organism '${use_pointfinder.use_pointfinder_condition}'
         #end if
 
         $named_genomes
     ]]></command>
     <inputs>
         <param type="data" name="genomes" format="fasta" multiple="true"/>
         <conditional name="use_pointfinder">
```

Comment:
You don't actually need this `<conditional>` section anymore. This was just for when we wanted to show/hide the drop-down menu when enabling/disabling pointfinder (see https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-inputs-conditional for more information).

Reply:
Removed it in my next commit